buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -84,6 +88,32 @@ class BuildPlugin implements Plugin<Project>  {
     }
 
     private static void configureConfigurations(Project project) {
+        if (project != project.rootProject) {
+            // Set up avenues for sharing source files between projects in order to create embedded Javadocs
+            // Import source configuration
+            Configuration sources = project.configurations.create("sources")
+            sources.canBeConsumed = false
+            sources.canBeResolved = true
```

Comment:
this is be false too ?  the sources shouldn't be resolved since they should only be declared as a single project reference ?

Reply:
Hmm, gradle does not seem to like this. In order for Gradle to convert the project dependency into actual files it needs to resolve the configuration. It needs to do this for the sources jar, and any time a file collection that contains the configuration is mutated.